src/typed-method-types/workflows/triggers/tests/trigger-workflow_test.ts

```diff
@@ -384,6 +386,136 @@ Deno.test("Trigger inputs are powered by generics", async (t) => {
       });
     });
 
+    await t.step("handles workflow with customizable inputs", async (t) => {
+      await t.step("allows customizable input to be set", async () => {
+        const _: Trigger<CustomizableInputWorkflow> = {
+          name: "TEST",
+          type: "shortcut",
+          workflow: "#/workflows/example",
+          inputs: { customizable: { customizable: true } },
+        };
```

Comment:
Q: I'm probably missing something but what's the point of this `const`? Ditto in all the other steps

Reply:
I assume Sam is just modeling this after what's done in the tests above, and we had those as separate tests because we'd see type errors on `client.workflows.triggers.create` that didn't happen when assigning the type to a value/variable (or vice versa).

We ended up fixing the root issue in our type system, but kept the seemingly redundant tests around in case someone else came by later and tried to make an incompatible change.